app/src/main/java/at/aau/serg/network/HttpClient.kt

```diff
@@ -34,6 +29,7 @@ class HttpClient private constructor(private val baseUrl: String) {
 
     fun get(url: String, authToken: String?, callback: Callback) {
         val requestUrl = makeRequestUrl(url)
+        print(requestUrl)
```

Comment:
I think you forgot to delete the print after testing

Reply:
Nice catch